.gitlab-ci.yml

```diff
@@ -15,6 +15,7 @@ stages:
   - build
   - publish_artifacts
   - pre_release
+  - pre_release_image
```

Comment:
I am adding the stage in this PR: https://github.com/vmware/versatile-data-kit/pull/631 

This also would change it so control service test are not triggered unnecssarily